### ISSUE TITLE
Compilation errors when submitting jar files on CodeCrunch 

### ISSUE THREAD

**Issue by winkmike** [nus-cs2030/2021-s2#48]
# Summary
I get compilation error `gcc: error trying to exec 'jc1': execvp: No such file or directory` when submitting jar files of Mock PA1 on CodeCrunch. I believe the error comes from submitting jar files instead of usual java files because it's the first time I have this issue.

## Description 
After a bit researching, it seems the cause can be:
1. lack package g++/gcc/jc1/... on the machine
2. problems relating to environment and path (GCC_EXEC_PREFIX)

I have tried the following codes in command prompt:
`sudo apt-get update`
`sudo apt-get install --reinstall build-essential`
`sudo apt-get install g++`
`sudo apt-get install gcc`
... (maybe some others but I don't remember all)
However, when I resubmitted CodeCrunch still give the same error.
I don't really understand environment and path so I haven't done anything with it.

By the way, these are the websites that I have tried finding information about the error:
[https://stackoverflow.com/questions/11912878/gcc-error-gcc-error-trying-to-exec-cc1-execvp-no-such-file-or-directory](url)
[https://gcc.gnu.org/onlinedocs/gcc/Environment-Variables.html](url)
[https://unix.stackexchange.com/questions/246443/gcc-error-trying-to-exec-cc1-execvp-no-such-file-or-directory](url)

Really appreciate if someone can help me resolve the problem! 

## Screenshots (if any): 
This is the screenshot of the Compilation Error on CodeCrunch
![2021-02-24_LI](https://user-images.githubusercontent.com/77266401/109030828-1e2dfb00-76ff-11eb-9925-ffcbf02c6a16.jpg)

**Comment by gableejh** [comment]
Not too sure how to get around this but perhaps you can submit it to the Level 4 task instead, named "CS2030 Mock Practical Assessment #1: Salary (Level 4)"

**Comment by winkmike** [comment]
Yes thank you! It work perfectly fine when I submitted normal java files in each level

**Comment by ydymegan** [comment]
Why not try submitting files one by one
